apko-build/action.yaml

```diff
@@ -7,6 +7,12 @@ description: |
   and tag to use and output a tar file and does not push to a registry.
 
 inputs:
+  apko-image:
+    description: |
+      The digest of the apko image to invoke.
+    default: ghcr.io/wolfi-dev/apko@sha256:3f7077f5a9d74747c03c03e18d202e81343e654f1b33220220af76db914dc1d1
```

Comment:
default to `:latest` ? see open pr #101

Reply:
Sure, this was just me staging things so that the pin could happen in our downstream repos.